src/components/NewQuestionnaire/Forms/QuestionDispose.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <Icon name="trash-can-outline" @click="deleteQuestion" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import Icon from '/@/components/UI/Icon.vue'
+
+export default defineComponent({
+  name: 'QuestionDispose',
+  components: {
+    Icon
+  },
+  props: {
+    questionarray: {
+      type: Array,
+      required: true
+    },
+    index: {
+      type: Number,
+      required: true
+    }
+  },
+  emits: {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    delete: (index: number) => true
+  },
+  setup(props, context) {
+    const deleteQuestion = () => {
+      const trash = confirm('この質問を削除しますか？')
```

Comment:
```suggestion
      const isTrash = confirm('この質問を削除しますか？')
```
好みレベルの修正ですが、真偽値が入ってる変数は`is`とかを先頭につけるとすっきりすることがあります(動詞だと関数感が出る)